packages/editor/src/editor/sidebar/fields/SelectField.tsx

```diff
@@ -3,24 +3,28 @@ import type { FieldOption } from '../../../types/config';
 
 type SelectFieldProps = {
   options: readonly FieldOption[];
-  label: string;
+  label?: string;
   value: string;
   onChange: (value: string) => void;
+  width?: string;
 };
 
-export const SelectField = ({ options, label, value, onChange }: SelectFieldProps) => (
-  <BasicField label={label}>
-    <Select value={value} onValueChange={change => onChange(change)}>
-      <SelectTrigger>
-        <SelectValue placeholder='Select an option' />
-      </SelectTrigger>
-      <SelectContent>
-        {options.map(option => (
-          <SelectItem key={`${option.value}`} value={option.value as string}>
-            {option.label}
-          </SelectItem>
-        ))}
-      </SelectContent>
-    </Select>
-  </BasicField>
-);
+export const SelectField = ({ options, label, value, onChange, width }: SelectFieldProps) => {
+  const style = width ? { width: width } : {};
```

Comment:
The dropdown is too small without a specified width, causing options to be cut off. This happens because the select element shares its width with the input fields and takes the width of the selected item. If the selected item is "equals to" and the dropdown is opened, the other options are truncated. Setting a width makes the dropdown wider so that all options are fully visible.
![grafik](https://github.com/user-attachments/assets/fa4b5928-6a9f-45b5-a987-46533c8e1cf4)

Reply:
Then isn't it easier to increase the width of the select itself? Because if you don't do it it will not have a consistent width over different conditions with different operators

Reply:
Yes, that is correct. However, I don’t think it’s a bad thing, as this way, the operator only takes up the width it needs, allowing for more space for the two input fields.
![grafik](https://github.com/user-attachments/assets/a19041ac-1b37-4a08-8fb7-773a5901630f)

If I set the width to the entire select, it would just look like this. What do you think looks better?
![grafik](https://github.com/user-attachments/assets/8b69db2c-5274-4b18-8617-4d4cd66d9d17)

Reply:
the thing isn't that the first thing is bad, my point is only that it makes the select component more complex because of the additional parameter. But I'm not strict against it.

Reply:
Ah, got it. Yes, I’m afraid I have to provide a width anyway since the select looks like it does in the first image due to the flex. However, I've simplified the component a bit more. 
Alternatively, if you prefer, I can create a CSS file for the Condition component and manage the width of the select through that without modifying the select element.

Reply:
provide this width via css will not be that easy as the menu is not below the trigger.
but yes I think you're right also the trigger needs the width as param

Reply:
Actually I believe the trigger doesn't even require a specified width. By using style={{ whiteSpace: 'nowrap' }}, I ensure that there is no word wrapping, allowing it to automatically adjust to the width necessary to display the entire content. The width is only needed for the SelectContent.